### ISSUE TITLE
Delete all broken symbolic links

### ISSUE THREAD

**Issue by oshanz** [bleachbit/wishlist#50]
there are broken links in
1. menu bar and sub menus
2. inside folders 

hope there is a way to clean those.

**Comment by az0** [comment]
What do you mean inside menus?

**Comment by oshanz** [comment]
I'm primary using Mate desktop environment. In main menu bar there are remaining links from uninstalled application. I can see same links when login from xfce or lxde sessions. haven't tried from unity or gnome shell. apt clean or purge doesn't solve it.

**Comment by Tobias-B-Besemer** [comment]
Think it would be nice to do this with an external EXE that is triggered by a CleanerML file.

**Comment by Tobias-B-Besemer** [comment]
Had just a short look, but this seems IMHO to look good: https://stackoverflow.com/questions/22097130/delete-all-broken-symbolic-links-with-a-line

**Comment by Tobias-B-Besemer** [comment]
@oshanz, for clearify:
Delete just Desktop/Start Menu broken links, or do a deep scan for all broken links in the system and delete them then?
I think two cleaners for each use case would be IMHO good...

**Comment by oshanz** [comment]
@Tobias-B-Besemer I also agree with that point. most annoying thing is broken menu links. if deep scan can take considerable time then its better to be two functions.
